packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -98,7 +139,7 @@ export interface EditableTree {
      * A mechanism for disambiguating this should be added,
      * likely involving an alternative mechanism for looking up global fields via symbols.
      */
-    readonly [key: string]: UnwrappedEditableField;
+    [key: string]: UnwrappedEditableField;
```

Comment:
Fixing the above TODO regarding global field keys by making the key type here FieldKey would be a good change to make in a separate PR before this one.

See MapTree as an example that handles this correctly.